site/content/docs/manifest/worker-service.en.md.md

```diff
@@ -0,0 +1,106 @@
+List of all available properties for a `'Worker Service'` manifest. To learn about Copilot services, see the [Services](../concepts/services.en.md) concept page.
+
+???+ note "Sample manifest for a worker service"
+
+    ```yaml
+    # Your service name will be used in naming your resources like log groups, ECS services, etc.
+    name: events-worker
+    type: Worker Service
+
+    # Your service is reachable at "http://api.${COPILOT_SERVICE_DISCOVERY_ENDPOINT}:8080" but is not public.
+
+    # Configuration for your containers and service.
+    image:
+      build: ./api/Dockerfile
+
+    cpu: 256
+    memory: 512
+    count: 1
+
+    subscribe:
+      topics:
+        - name: events
+          service: api
+          queue:
+            timeout: 10s
+            dead_letter:
+              tries: 5
```

Comment:
What do you think of removing this? I kinda want to push folks to just use the shared queue and only surface this if they reach out to us 💭